File: Area Amministratore/GiocoDislessia/src/model/Frase.java
package model;

public class Frase {
	private int id;
	private int livello;
	
	public Frase(int id, int livello)
	{
		this.id=id;
		this.livello=livello;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public int getLivello() {
		return livello;
	}

	public void setLivello(int livello) {
		this.livello = livello;
	}

	
	
}

File: Gioco in Unity/GiocoDislessiaRemotoServlet/Assets/Bean/Frase.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class Frase
{
    private int id;
    private List<Parole> parole;

    public Frase(int id)
    {
        this.id = id;
        parole = null;
    }

    public int getId()
    {
        return id;
    }

    public void setId(int id)
    {
        this.id = id;
    }

    public void setParole(List<Parole> parole)
    {
        this.parole = parole;
    }

    public List<Parole> getParole()
    {
        return parole;
    }

    public void addParole(Parole parola)
    {
        if(parole == null)
        {
            parole = new List<Parole>();
        }
        parole.Add(parola);
    }
}
